tfjs-core/src/ops/expand_dims_test.ts

```diff
@@ -34,7 +34,7 @@ describeWithFlags('expandDims', ALL_ENVS, () => {
   it('1d, axis=-3', () => {
     expect(() => {
       tf.tensor1d([1, 2, 3]).expandDims(-3);
-    }).toThrowError('Axis must be in the interval [-2, 1]');
+    }).toThrowError();
```

Comment:
Looks like GH didn't save my comment here. Was also curious about this change to the test? I'm guessing its because node throws a different error but just want to double check. Or maybe node does not thrown any error?